tools/envoy_headersplit/code_corpus/fail_mocks.h

```diff
@@ -0,0 +1,703 @@
+#pragma once
```

Comment:
For future tests, it'd be better to use the smallest representative set of testdata. Using 15 large mocks in the test header makes test runtime longer and debugging more difficult; if 3 smaller mock classes would suffice to exercise all the code under test, it's helpful to future engineers to reduce test input size.

Of course, some tests need large test input to fully exercise the code under test, and that's okay - but I think that isn't the case here.

Reply:
this test has been reduced to smaller case (2 mock class) to make libclang fail in parsing.